src/app/components/ResultsCount/ResultsCount.jsx

```diff
@@ -67,7 +88,16 @@ class ResultsCount extends React.Component {
     if (count !== 0) {
       return (<h2>Displaying {currentResultDisplay} of {countF} results {displayContext}</h2>);
     }
-    return (<h2>No results found. Please try another search.</h2>);
+
+    if (this.checkSelectedFilters()) {
+      return (
+        <h2>No results for the keyword "{searchKeywords}" with the chosen filters. Try
+          a different search or different filters.
+        </h2>
+      );
+    }
+
+    return (<h2>No results for the keyword "{searchKeywords}". Try a different search.</h2>);
```

Comment:
should we have a limit here for the length of the keywords?

Reply:
No.